README.md

```diff
@@ -1 +1,38 @@
-# hw01
\ No newline at end of file
+
+# My Autobiography
+
+
+My name is Tatiana Hernández and I am from Ohio. Currently, I am a graduate student in the MAPSS program. Some of my research interests include:
+
+* *Central American* emigration
+* state criminalization of *deportees*
+* public *perceptions* of violence and punitive anticrime measures
+
+I am especially interested in studying political topics related to **El Salvador**:
+
+![Map of El Salvador](https://www.nationsonline.org/maps/el_salvador_map.jpg)
+
+My dad is from El Salvador and my grandma and uncle live in the city of **San Salvador**. You can find San Salvador close to **Lago de Ilopango** on the map. I also have an aunt and uncle that live in South Korea. I was able to get to know them better after college because I lived there for three years.
+
+![Anyang](https://upload.wikimedia.org/wikipedia/commons/f/f0/Anyang_city.jpg)
+
+This is Anyang-si, where I used to live. I loved my neighborhood and I cannot wait to visit it again. In the future, I would also like to travel to more places.
+
+
+*Here are a few places I have been:*
+
+1. Bolivia
```

Comment:
I would suggest adding a header here instead, since it may be hard to differentiate between sections otherwise